Remove old code editor and active learning from HTML images (#39868)
mdn/content: files/en-us/learn_web_development/core/structuring_content/html_images/index.md

It is better to include such supporting information in the main article text, rather than attached to the image.

### Image embedding practice

It is now your turn to play! This task will get you to embed an image.

1. Click **"Play"** in the code block below to edit the example in the MDN Playground.
2. Use the provided {{htmlelement("img")}} tag to embed the image located at the following URL:

   ```url
   https://raw.githubusercontent.com/mdn/learning-area/master/html/multimedia-and-embedding/images-in-html/dinosaur_small.jpg
   ```

   > [!NOTE]
   > Earlier we said to never hotlink to images on other servers without permission, but this image is on our GitHub repo, so it is OK.

> 3. Add some `alt` text to your image, and check that it works by misspelling the image URL.

Reviewer: ```suggestion
3. Add an `alt` text for the image. You can check that the alt text works by temporarily misspelling the image URL.
```
